src/scripts/components/globe-navigation/globe-navigation.tsx

```diff
@@ -17,6 +18,7 @@ const GlobeNavigation: FunctionComponent = () => {
   const dispatch = useDispatch();
   const defaultView = config.globe.view;
   const projectionState = useSelector(projectionSelector);
+  const globecheck = useSelector(globeViewSelector);
```

Comment:
If we use the position from the redux store we only get a new value when the globe movement ends. It would be cooler if the icon rotation updates in realtime. Perhaps we can do a little trick and set the rotation as a css variable in the globe component.